source/connection-monitoring-and-pooling/tests/pool-clear-schedule-run.yml

```diff
@@ -0,0 +1,46 @@
+version: 1
+style: unit
+description: Pool clear SHOULD schedule the next background thread run immediately
```

Comment:
If we go with my suggestion above, then I think it makes sense to rename this test to "Connections MUST be interrupted as soon as possible (interruptInUseConnections=true)", and we can rename the test file `pool-clear-interrupt-immediately.yml` or something like that. The rest of this test can stay the same. The other test of the background thread scheduling behavior for when interruptInUseConnections=false can stay as-is.

Reply:
done